apps/web/src/server/app/lpa/lpa.controller.js

```diff
@@ -26,3 +25,154 @@ export async function getLpaDashboard(request, response, next) {
 		questionnairesList
 	});
 }
+
+/**
+ * GET the questionnaire review page
+ * It will fetch the questionnaire details for the specified ID and will render the review form.
+ *
+ * @param {import('express').Request} request - Express request object
+ * @param {import('express').Response} response - Express request object
+ * @param {Function} next  - Express function that calls then next middleware in the stack
+ * @returns {void}
+ */
+export async function getReviewQuestionnaire(request, response, next) {
```

Comment:
Just leave this as is, as I'll likely update all request handler signatures to a stronger-typed custom declaration at some point, but for info, a simpler way to js-document this function is:

/** @type {import('express').RequestHandler} */

(It also accepts generics)